.github/workflows/build_container.yml

```diff
@@ -121,3 +119,15 @@ jobs:
             mondoo/client:${{ steps.semver.outputs.major }}-ubi-rootless
             mondoo/client:latest-ubi-rootless
 
+  package-macos:
+    name: Run macOS Universal pkg Release workfow
+    runs-on: ubuntu-latest
+    needs: build_container
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: Run macOS Universal pkg Release workfow
+        env:
+          GH_TOKEN: ${{ secrets.REPO_API_TOKEN }}
+        run: gh workflow run pkg_macos.yaml
```

Comment:
We should call flows using peter-evans/repository-dispatch@v3 to be consistent with how we do it elsewhere.

Reply:
I accept this, it is much cleaner, just not consistent.  We should perhaps adopt this method going forward elsewhere.